src/GameLoop.tsx

```diff
@@ -377,7 +388,24 @@ export function GameLoop() {
     }
 
     function handleStates() {
-        if ("spawningBlock" === stateHandler.state.value) {
+        if ("startingGame" === stateHandler.state.value) {
+            setCountdownVisibility(true);
+            setCountdownStartTime(performance.now());
+            stateHandler.send("START");
+        }
+        else if ("countdown" === stateHandler.state.value) {
+            countdownTimeElapsed = performance.now() - countdownStartTime;
+            let index: number = countdownTotalSteps - Math.floor(countdownTimeElapsed / 1000);
```

Comment:
```suggestion
            let index = countdownTotalSteps - Math.floor(countdownTimeElapsed / 1000);
```

Don't need type annotations where it can be inferred.

Reply:
Done (renamed index to `currCountdownNum`)